testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/input/MiniAppInputActivity.kt

```diff
@@ -1,23 +1,34 @@
 package com.rakuten.tech.mobile.testapp.ui.input
 
+import android.content.Context
+import android.content.Intent
 import android.os.Bundle
-import android.view.View
-import androidx.appcompat.app.AppCompatActivity
 import com.google.android.material.textfield.TextInputEditText
 import com.rakuten.tech.mobile.miniapp.testapp.R
+import com.rakuten.tech.mobile.testapp.ui.base.BaseActivity
 import com.rakuten.tech.mobile.testapp.ui.display.MiniAppDisplayActivity
+import com.rakuten.tech.mobile.testapp.ui.miniapplist.MiniAppListActivity
 import kotlinx.android.synthetic.main.mini_app_input_activity.*
 
-class MiniAppInputActivity : AppCompatActivity() {
+class MiniAppInputActivity : BaseActivity() {
+
+    companion object {
+        fun start(context: Context) {
```

Comment:
Any specific reason for placing the launch-logic within?

Reply:
Yeah. There is a restriction here when launching this activity: `addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP or Intent.FLAG_ACTIVITY_NEW_TASK)`

Reply:
I noticed another one in the list activity. How about extracting them into a unit like `ActivityLauncher.kt`. What do you think?

Reply:
I think just keep it separately as it is now. As specific activity should have its own launching method.